spec/features/admin/configuration/dynamic_content_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+
+# Defines the test link string
+TEST_LINK = "ofn test link"
+
+# Defines the test href string. This one should contain all non-encoding characters, for a URL
+TEST_HREF = ":/?#@!$&'()*+,;=0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+
+describe 'add dynamic content to homepage' do
+	include AuthenticationHelper
```

Comment:
you must use 2 spaces for indentation, looks like you may be using tabs, this is something you configure in your IDE. are you using sublimetext?

Reply:
Yes, I am - I've configured tab to insert two spaces on Sublime, but somehow rubocop complains and detects tabs here. Anyway, to work around by adding to spaces manually :+1: